Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.23 AS builder
+
+RUN adduser \
+  --disabled-password \
+  --gecos "" \
+  --home "/nonexistent" \
+  --shell "/sbin/nologin" \
+  --no-create-home \
+  --uid 65532 \
+  default-user
+
+WORKDIR $GOPATH/app/
+
+COPY . .
+
+RUN go mod download
+RUN go mod verify
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /main ./cmd/bff
+
+FROM scratch
+
+COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /etc/passwd /etc/passwd
+COPY --from=builder /etc/group /etc/group
```

Comment:
@shan-96
1) here just lack of my knowledge do we need zoneinfo, passwd and group? what will be broken if we don't copy that? 
2) shall we copy default config with no defined calls and ship it in image by default? to be able to start server in pass-trough mode by default. now it'll be crashing if we don't mount config to the container

Reply:
1. yes we do not need to create default user. I can remove it. Probably even the certs
2. sure, but lets handle that in docker-compose